src/openshift/openshift-pipeline-template.yml

```diff
@@ -36,29 +36,15 @@ parameters:
 
 ##
 # Objects created by the template:
-# * An image stream containing the application
 # * A build config using the pipeline strategy
-# * A deployment config / replication controller instantiating 1 instance of the application
-# * A service to make the application available to other application running in OpenShift
-# * A route making the application accessible from outside OpenShift
```

Comment:
Ahh I see. My bad then. Sorry I did not track this change. I was working with pipeline and found mention of this file in readme.